### PR TITLE
Create meeting step 3

### PR DESCRIPTION
**Pull request by anna-giang**
Prototype for manual input of proposed times for the meeting - Step 3 of the create meeting sequence.

### CODE REVIEW

**Comment by anna-giang** [comment]
Rebased the branch onto `main` before submitting pull request, hence the 'duplicated' commits.

**Comment by anna-giang** [comment]
Screen Shot 2021-01-12 at 9 15 56 am

Screenshot of the added HTML page `create-meeting-step-3.html` 😊

**Review by theresenieva — COMMENTED** [review]
Yup, lgtm

**Review by vivianho18 — COMMENTED** [review]
lgtm!

File: prototype/src/main/webapp/createMeeting.js [hunk 1]
```diff
@@ -35,3 +37,47 @@ function addMeetingGuest() {
...
+ * Add a meeting time datetime-local input to the page, enclosed in 
+ * a 'li' element.
+ * The element appended to the DOM will have an id of 'meeting-time-NUMBER', 
+ * where NUMBER equal to value of MEETING_TIMES_ADDED (which is incremented
+ * each time this function is called). This is to identify the meeting time input
+ * so that users may remove the meeting time. See removeMeetingTime().
+ */
+function addMeetingTime() {
+  let listItem = document.createElement('li');
+  listItem.id = 'meeting-time-' + MEETING_TIMES_ADDED;
```
Comment (ghost) [inline prototype/src/main/webapp/createMeeting.js]: Did you mean `meetingTimesAdded`? Same for all the other occurences of this.

File: prototype/src/main/webapp/createMeeting.js [hunk 2]
```diff
@@ -35,3 +37,47 @@ function addMeetingGuest() {
...
+  // delete button associated with this meeting time
+  let removeTimeButton = document.createElement('button');
+  removeTimeButton.innerHTML = '-';
+  removeTimeButton.addEventListener('click', function() {removeMeetingTime(listItem.id);});
+
+  listItem.appendChild(newTimeInputLabel);
+  listItem.appendChild(newTimeInput);
+  listItem.appendChild(removeTimeButton);
+
+  let timeListDiv = document.getElementById('meeting-time-list')
```
Comment (ghost) [inline prototype/src/main/webapp/createMeeting.js]: nit: semicolon

**Review by ghost — COMMENTED** [review]
Something to think about is that the user may want to delete the first time input when there are more than one. For the prototype this is probably fine, but think about how you could show the remove buttons when there are multiple inputs, and then hide them when there is only one.

File: prototype/src/main/webapp/createMeeting.js (hunk 1 above)
Comment (anna-giang) [reply prototype/src/main/webapp/createMeeting.js]: Oh no, I forgot to change it everywhere 😅 I've changed it now, thanks for picking that up!

**Review by ghost — APPROVED** [review]

**Pull request merged**